### DISCUSSION TITLE
Support for Ollama

### DISCUSSION THREAD

**Discussion by ethanannane** [RayVentura/ShortGPT#136] [Ideas]
This is an awesome project, but unfortunately, I am not able or willing to use it due to its requirement of an Open-AI API key. I might have a look myself and see what I can do to get it working, but it would be good if other people could discuss this. I am planning to just have a server grinding out stuff when I ask it, and would like to make everything local, and this is the big problem with this idea. I also think support for a text file as a script would also be beneficial Thank You!!

**Comment by ethanannane** [comment]
Also if someone could point me to the script that deals with script gen, please help, I am lost in this codebase

**Reply by alaughingpillow** [reply]
from what i could see its in the gpt folder eveything there is gpt related or you could just go to vs code and search for open ai and see from there
